ProjFS.Mac/PrjFSLib/prjfs-log/prjfs-log.cpp

```diff
@@ -79,14 +79,8 @@ static void ProcessLogMessagesOnConnection(io_connect_t connection, io_service_t
     dispatch_source_set_event_handler(logState->dataQueue.dispatchSource, ^{
         DataQueue_ClearMachNotification(logState->dataQueue.notificationPort);
         
-        while(true)
+        while(IODataQueueEntry* entry = DataQueue_Peek(logState->dataQueue.queueMemory))
```

Comment:
I *think* this is unrelated code cleanup.  As a general rule I like new PRs for that sort of stuff and am happy to jump on those quickly!

Also to double check, entry = nullptr will break the loop

Reply:
> I think this is unrelated code cleanup. As a general rule I like new PRs for that sort of stuff and am happy to jump on those quickly!

Yep it's code cleanup and I am generally in the same boat 😄 

In this instance I was getting confused over why the main loop of prjfs-log and the logging daemon looked so different.  After determining they were doing the same thing I updated prjfs-log to use the same pattern as the daemon, please let me know if you'd rather I back that change out.

Reply:
I wouldn't bother backing it out.  Maybe do a tiny separate PR next time, but not worth the trouble here.

Reply:
>Maybe do a tiny separate PR next time, but not worth the trouble here.

Will do, thanks!